### DISCUSSION TITLE
Missing privacy manifest

### DISCUSSION THREAD

**Discussion by emmaB93** [capawesome-team/capacitor-mlkit#149] [Q&A] [answered]
I updated the barcode dependency to the newest version

`"@capacitor-mlkit/barcode-scanning": "6.0.0",`

After building the ios project and uploading it I always get the warning from apple that the privacy manifest is missing. Also with privacy_manifest_analyser.sh I'm getting the messages:

`Analyzing MLKitBarcodeScanning ...
⚠️  Missing privacy manifest file!
API usage analysis result(s): 2

🛠️  Descriptions for the following required API reason(s) may be missing: 2
[0] NSPrivacyAccessedAPICategoryFileTimestamp
[1] NSPrivacyAccessedAPICategorySystemBootTime`

Do I have to add the manifest manually or what is the correct solution here?

**Comment by robingenz** [comment, marked as answer]
> Do I have to add the manifest manually or what is the correct solution here?

No, the privacy manifest should be included in the ML Kit SDK. Seems like they released a new version last week: https://github.com/googlesamples/mlkit/issues/765
I will update the dependencies and publish a new plugin version (see #150).
